### PR TITLE
Make IonQ native gate docstring format correctly

### PR DESCRIPTION
**Pull request by dabacon**

### CODE REVIEW

**Comment by dabacon** [comment]
@Cynocracy

File: cirq-ionq/cirq_ionq/ionq_native_gates.py [hunk 1]
```diff
@@ -67,22 +74,35 @@ def _circuit_diagram_info_(
...
-        [[0, e^{-i*2*\pi*\phi}],
-         [e^{i*2*\pi*\phi}, 0]]
-    It is driven by Rabi laser.
-    https://ionq.com/best-practices
+    r"""An instance of the single qubit GPI gate with no phase.
+
+    The unitary matrix of this gate is:
+    $$
+    \begin{bmatrix}
+      0 & 1 \\
```
Comment (Cynocracy) [inline cirq-ionq/cirq_ionq/ionq_native_gates.py:82]: Ah, this is meant to document the null instance? neat

File: cirq-ionq/cirq_ionq/ionq_native_gates.py [hunk 2]
```diff
@@ -67,22 +74,35 @@ def _circuit_diagram_info_(
...
+      1 & 0
+    \end{bmatrix}
+    $$
+
+    See [IonQ best practices](https://ionq.com/best-practices){:external}.
```
Comment (Cynocracy) [inline cirq-ionq/cirq_ionq/ionq_native_gates.py:87]: If you don't mind changing them, these can be https://ionq.com/docs/getting-started-with-native-gates now

**Review by Cynocracy — APPROVED** [review]
The unitaries for the document() strings are pretty and I feel like I learned a bit more about the gates by reading them :)

File: cirq-ionq/cirq_ionq/ionq_native_gates.py (hunk 1 above)
Comment (dabacon) [reply cirq-ionq/cirq_ionq/ionq_native_gates.py:82]: yeah, docs constants.

File: cirq-ionq/cirq_ionq/ionq_native_gates.py (hunk 2 above)
Comment (dabacon) [reply cirq-ionq/cirq_ionq/ionq_native_gates.py]: Done

**Review by gmauricio — APPROVED** [review]

**Pull request merged**